ansible/script/sdsrc

```diff
@@ -37,4 +37,4 @@ DEV_STACK_DIR=$STACK_HOME/devstack
 
 GOPATH=${GOPATH:-$HOME/gopath}
 OPENSDS_DIR=${GOPATH}/src/github.com/opensds/opensds
-
+ANSIBLE_CONF_DIR=${HOME}/opensds-installer/conf
```

Comment:
How can you be suer that the `opensds-installer` project is under `$HOME`? It should not be hard-coded here.